dev-requirements.txt

```diff
@@ -471,7 +471,7 @@ six==1.16.0
     #   python-dateutil
 sniffio==1.3.1
     # via asgi-tools
-superdesk-core @ git+https://github.com/superdesk/superdesk-core.git@async
+superdesk-core @ git+https://github.com/superdesk/superdesk-core.git@hg/NHUB-534-cacheable-service
```

Comment:
Updated this just to make sure tests pass. I'll fix it back to async once this PR https://github.com/superdesk/superdesk-core/pull/2688 gets merged.